src/exts/tag.py

```diff
@@ -101,57 +102,35 @@ async def list(self, ctx: interactions.CommandContext):
 
         log.debug("Matched for list. Returning result...")
 
-        embed = interactions.Embed(
-            title="Tags list",
-            description="This is the current list of existing tags.",
-            color=0x5865F2,
-            fields=[
-                interactions.EmbedField(
-                    name="Names",
-                    value=" ",
-                    inline=True,
-                ),
-                interactions.EmbedField(
-                    name="(cont.)",
-                    value=" ",
-                    inline=True,
-                ),
-            ],
-        )
-        _last_name: str = ""
         _id: int = 0
+        _pages: list = []
+        _contents: list = []
+        _embeds: list = []
+        _value: str = ""
 
-        for tag in db:
-            if (len(tag) + len(embed.fields[0].value)) < 900:
-                embed.fields[0].value += f"` {_id} ` {tag}"
-            else:
-                break
+        def _divide(_list):
+            for i in range(0, len(_list), 5):
+                yield _list[i : i + 5]
 
-            _last_name = tag
+        for tag in db:
+            _value += f"` {_id} ` {tag}\n"
             _id += 1
 
-        if _id < len(db):
-            ...
+        for _v in _divide(_value.split("\n")):
```

Comment:
Use a more descriptive variable than `_v`

(no just value doesn't count because I still have no clue what it does maybe you should use a comment lol)

Reply:
Can you suggest changing it to `tag_group` or smth. Each one should be a list of 5 tags but I kinda forgot what comes after that.